kubernetes-popups.el

```diff
@@ -56,16 +56,14 @@
     (?v "Exec into container using vterm" kubernetes-exec-using-vterm))
   :default-action 'kubernetes-exec-into)
 
-(magit-define-popup kubernetes-file-popup
-  "Popup console for file commands for POD."
-  :group 'kubernetes
-  :options
-  '("Options for customizing file behaviour"
-    (?c "Container" "--container=" kubernetes-utils-read-container-name))
-  :actions
-  '((?f "Find file" kubernetes-tramp-find-file)
-    (?d "Dired" kubernetes-tramp-dired))
-  :default-action 'kubernetes-tramp-dired)
+(transient-define-prefix kubernetes-file ()
+  "Work with files in Kubernetes resources."
+  [["Options"
+    ;; TODO: This doesn't currently get picked up by the suffixes
+    ("=c" "Container" "--container=" kubernetes-utils-read-container-name)]]
```

Comment:
Shouldn't we fix this before mergeing?

Reply:
To be clear, it does not appear that either `kubernetes-tramp-dired` or `kubernetes-tramp-find-file` pick up the container name from the popup options in `master` currently. So I would argue that we're no better or worse off than before w/ this PR, giving us the opportunity to address this TODO the Transient way after merging this PR.

Let me know your thoughts here. I'm also happy to fix this issue before switching this popup to Transient, in which case I'll open a separate PR w/ that fix.

Reply:
ok

Reply:
See #205 👍